Patch-Source: https://github.com/stha09/chromium-patches/blob/cd45403f83a2305d388e1b1f930ca05c13535636/chromium-105-Bitmap-include.patch
From 8959d9d1c6e4ed083c278ab9e8def20a409052b9 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 22 Jul 2022 16:51:28 +0000
Subject: [PATCH] IWYU: add memory for std::unique_ptr in disk_cache::Bitmap

Bug: 957519
Change-Id: I123198345e5f9062329b7eabe980f312525c268b
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3779530
Reviewed-by: Maks Orlovich <morlovich@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#1027289}
---

diff --git a/net/disk_cache/blockfile/bitmap.h b/net/disk_cache/blockfile/bitmap.h
index 07806cf..9ffa98b9 100644
--- a/net/disk_cache/blockfile/bitmap.h
+++ b/net/disk_cache/blockfile/bitmap.h
@@ -8,6 +8,8 @@
 #include <stdint.h>
 #include <string.h>
 
+#include <memory>
+
 #include "base/memory/raw_ptr.h"
 #include "net/base/net_export.h"
 
